src/Backend/SgateShopgatePlugin/Components/Export.php

```diff
@@ -31,6 +31,8 @@ class Shopware_Plugins_Backend_SgateShopgatePlugin_Components_Export
     const CACHE_KEY_CUSTOMERGROUPS = 'customer_groups_ids';
     const CACHE_KEY_CATEGORY_PRODUCT_SORTING = 'categories_product_sort_';
     const CACHE_KEY_STREAM_CATEGORY_PRODUCT_SORTING = 'stream_categories_product_sort_';
+    const CACHE_TIME_IN_SECONDS = 28800;
```

Comment:
Why cache time? We talked about an import instead of time based cache. If you cannot access the import ID use offset == 0 parameter as condition to flush the cache.